org.civicrm.reqangular/reqangular.php

```diff
@@ -107,6 +107,18 @@ function reqangular_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
   _reqangular_civix_civicrm_alterSettingsFolders($metaDataFolders);
 }
 
+/**
+ * Implements hook_civicrm_buildForm().
+ *
+ * @param string $formName
+ * @param CRM_Core_Form $form
+ */
+function reqangular_civicrm_buildForm($formName, &$form) {
+  if(_reqangular_checkIfFormRequiresReqangular($formName)) {
```

Comment:
maybe `_reqangular_doesFormRequiresReqangular` is better

Reply:
Its not required anymore.